core/services/gateway/handlers/functions/subscriptions/orm.go

```diff
@@ -97,28 +98,33 @@ func (o *orm) UpsertSubscription(subscription CachedSubscription, qopts ...pg.QO
 		subscription.BlockedBalance = big.NewInt(0)
 	}
 
+	var consumers [][]byte
+	for _, c := range subscription.Consumers {
+		consumers = append(consumers, c.Bytes())
+	}
```

Comment:
Checking the logs while deploying the local DON I saw errors of the type:
`cannot convert 0x2Ff71d2c4eF7FffaC12B73993ac36260437066aA to Bytea in ByteaArray`

I was not catching these errors in the unit tests because all examples had an empty consumer array, so I modified the test and was able to reproduce the issue. Postgres does not seem to handle the conversion from slice to array of bytes correctly and checking the code looks like the correct way of storing this is doing this conversion before. The logpuller is doing this for `[]common.Hash` [here](https://github.com/smartcontractkit/chainlink/blob/develop/core/chains/evm/logpoller/log_poller.go#L632)

Reply:
Sounds good. Thanks for the explaination!

Reply:
Also, thanks for adding a test to cover this. Great habits!